doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1744,6 +1744,12 @@ Number notations
          sorts, primitive integers, primitive floats, primitive arrays and type
          constants for primitive types) will be considered for printing.
 
+         .. note::
+            As a particular case, :n:`@qualid__type` can be :n:`PrimInt63.int`,
+            in which case :n:`@qualid__print` takes as input :n:`PrimInt63.int_wrapper`
+            instead of :n:`PrimInt63.int`. See below for an
```

Comment:
```suggestion
            For example, :n:`@qualid__type` can be :n:`PrimInt63.int`,
            in which case :n:`@qualid__print` takes :n:`PrimInt63.int_wrapper` as input
            instead of :n:`PrimInt63.int`. See below for an
```